backend/src/server/routes/v1/organization-router.ts

```diff
@@ -68,17 +72,29 @@ export const registerOrgRouter = async (server: FastifyZodProvider) => {
     },
     onRequest: verifyAuth([AuthMode.JWT]),
     handler: async (req) => {
-      const users = await server.services.org.findAllOrgMembers(req.permission.id, req.params.organizationId);
+      const users = await server.services.org.findAllOrgMembers(
+        req.permission.id,
+        req.params.organizationId,
+        req.permission.orgId
+      );
       return { users };
     }
   });
 
   server.route({
     method: "PATCH",
-    url: "/:organizationId/name",
+    url: "/:organizationId",
     schema: {
       params: z.object({ organizationId: z.string().trim() }),
-      body: z.object({ name: z.string().trim() }),
+      body: z.object({
+        name: z.string().trim().optional(),
+        slug: z
+          .string()
+          .trim()
+          .regex(/^[a-zA-Z0-9-]+$/, "Name must only contain alphanumeric characters or hyphens")
+          .optional(),
+        authEnforced: z.boolean().optional()
```

Comment:
I'm not sure I understand your point. Can you elaborate?

The idea here is that permissioned users should be able to make a `PATCH` request to enforce/un-enforce org-level authentication for a particular organization; they can switch it on or off.

It is fine if SAML is **active** but `authEnforced=false`; this is intentional and would be usually the case now when setting up SAML. When you first set up SAML:

- The initial state is `false`.
- You configure SAML.
- You activate SAML that is to set its active state to `true` (this will allow users to now log into Infisical with SAML).
- After a successful login is registered with SAML, then admins can hit the toggle to enforce SAML auth for the organization.

This is to prevent users from providing an invalid SAML configuration and accidentally getting locked out; you must now show that you've successfully authenticated with SAML once in order to enforce it for the organization.